inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -66,4 +69,38 @@ void ConvolutionLayerTest::SetUp() {
     ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(conv)};
     function = std::make_shared<ngraph::Function>(results, params, "convolution");
 }
+
+void ConvolutionLayerTest::GenerateInputs() {
+    const auto& inputsInfo = executableNetwork.GetInputsInfo();
```

Comment:
should not be redefined here

Reply:
Yes, it is fixed to use LayerTestsCommon::GenerateInputs.